vivisect/tests/linux_amd64_libstdc_data.py

```diff
@@ -7451,9 +7451,8 @@
         ('libstdc___so_6_0', 0x383b30, 2, 0x37c3c8),
     ],
 
-    'names' : [
+    'names': [
         (0x2088480, 'libstdc___so_6_0.init_function'),
-        (0x20884a0, 'libstdc___so_6_0.LazyLoaderTrampoline'),
```

Comment:
i removed LazyLoader but was thinking about adding it back in.  it's just another step and i wasn't sure if it was super valuable.  thoughts?

Reply:
as for the _0's... because PLT's *must* be named, and they still need to be unique, to avoid having to plaster a va into every PLT (which makes analysis a pain), we allow multiples to get a _# tagged to the end.  this is based on the order they are identified, which means that this branch changed the order they are discovered (big surprise).